include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -130,7 +139,15 @@ enum class sweep_parameter
      */
     MU_MINUS
 };
-
+namespace detail
+{
+/**
+ * Forward-declaration for `operational_domain`.
```

Comment:
@marcelwa Do we want to add ``@tparam`` and ``@param``?

Reply:
No, this is just a forward-declaration and doesn't appear in the docs.